File: api/DTOs/Admin/OfferConclusionDto.cs
namespace api.DTOs.Admin
{
    public class OfferConclusionDto
    {
        public int EmploymentId { get; set; }
        public string AcceptedString { get; set; }      //accepted or rejected
        public bool OfferAccepted { get; set; }
        public string ConclusionStatus { get; set; }
        public DateTime ConclusionDate { get; set; }
    }
}

File: client/src/app/_dtos/admin/offerConclusionDto.ts
export interface IOfferConclusioDto
{
    employmentId: number;
    acceptedString: string;
    conclusionDate: Date;
}

export class OfferConclusioDto implements IOfferConclusioDto
{
    employmentId = 0;
    acceptedString = '';
    conclusionDate= new Date();
}